hosted-ce/30-remote-site-setup.sh

```diff
@@ -164,6 +164,10 @@ fi
 # Add the ability for admins to override the default Bosco tarball URL (SOFTWARE-4537)
 [[ $BOSCO_TARBALL_URL ]] && bosco_cluster_opts+=(--url "$BOSCO_TARBALL_URL")
 
+if ! fgrep -- '--copy-ssh-key' /usr/bin/bosco_cluster; then
```

Comment:
Do you want the line containing `--copy-ssh-key` to be printed on stdout?  (that's what happens here currently)

If you just want a simple test, rather than printing the matching lines, you can add the `-q` option to your fgrep line, eg

```suggestion
if ! fgrep -q -- '--copy-ssh-key' /usr/bin/bosco_cluster; then
```

Reply:
I think dumping to stdout is potentially useful here if operators need to troubleshoot CE startup